pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -67,6 +69,10 @@ func NewManager(
 	recorder record.EventRecorder,
 	nodeRef *api.ObjectReference,
 	clock util.Clock) (Manager, lifecycle.PodAdmitHandler, error) {
+
+	// TODO: pass this into the manager
+	withImageFs := false
```

Comment:
it's actually not needed on managerImpl once we build the ranking funcs.

I will leave it where it is now, and just pass it into NewManager in follow-up.